db/fixtures/miq_product_features.yml

```diff
@@ -5127,3 +5127,29 @@
     :description: Delete Arbitration Settings
     :feature_type: admin
     :identifier: delete_arbitration_setting
+
+# ArbitrationRules
+- :name: Arbitration Rules
+  :description: All functions for Arbitration Rules
+  :feature_type: admin
+  :identifier: miq_arbitration_rules
+  :children:
+  - :name: Show Arbitration List
+    :description: Show Arbitration Rule list
+    :feature_type: admin
+    :identifier: arbitration_rule_show_list
+  - :name: Create Arbitration Rule
+    :description: Create Arbitration Rule
+    :feature_type: admin
+    :identifier: arbitration_rule_new
+  - :name: Delete Arbitration Rule
+    :description: Delete Arbitration Rule
+    :feature_type: admin
+    :identifier: arbitration_rule_delete
+  - :name: Edit Arbitration Rule
+    :description: Edit Arbitration Rule
+    :feature_type: admin
+    :identifier: arbitration_rule_edit
+
```

Comment:
@imtayadeway @jntullo I think it is ok to support any expressions as long as they are going through RBAC in the context of a user when they are evaluated.